src/toil/test/batchSystems/batchSystemTest.py

```
@@ -179,7 +182,7 @@ def testSetEnv(self):
             # invoke that script rather than inline the test via -c.
             def assertEnv():
                 import os, sys
-                sys.exit(0 if os.getenv('FOO') == 'bar' else 42)
+                sys.exit(23 if os.getenv('FOO') == 'bar' else 42)
```

Changed to make it easier to detect the true result versus a masked error